src/network/mcpe/NetworkSession.php

```diff
@@ -806,8 +806,7 @@ public function syncAbilities(Player $for) : void{
 			$isOp ? PlayerPermissions::OPERATOR : PlayerPermissions::MEMBER,
 			$for->getId(),
 			[
-				//TODO: dynamic flying speed! FINALLY!!!!!!!!!!!!!!!!!
-				new UpdateAbilitiesPacketLayer(UpdateAbilitiesPacketLayer::LAYER_BASE, $boolAbilities, 0.05, 0.1),
+				new UpdateAbilitiesPacketLayer(UpdateAbilitiesPacketLayer::LAYER_BASE, $boolAbilities, $for->getFlySpeed(), $for->getWalkSpeed()),
```

Comment:
`getWalkSpeed()` doesn't exist. Please revert it to its original value, since we're not going to use it.